fah_alchemy/tests/integration/interface/client/test_client.py

```diff
@@ -51,3 +51,11 @@ def test_create_network(
         uvicorn_server,
     ):
         ...
+
+    def test_api_check(
```

Comment:
Can you add this to `fah_alchemy/tests/integration/compute/client/test_compute_client.py` as well?